legal-api/src/legal_api/resources/v2/admin/configuration.py

```diff
@@ -38,7 +37,7 @@ def get_configurations():
     }), HTTPStatus.OK
 
 
-@bp.route('', methods=['PUT'])
+@bp_admin.route('/configurations', methods=['PUT'])
 @cross_origin(origin='*')
 @jwt.has_one_of_roles([UserRoles.staff])
```

Comment:
@argush3 any staff can update configuration?

Reply:
sure, gonna confirm with Dylan, will keep you posted

Reply:
we have a role `UserRoles.admin_edit`, which we use in admin page. This will be a staff with an additional admin_edit role (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/resources/v2/administrative_bn.py#L34)

Reply:
Dylan just confirmed that any staff can update dissolution configuration